src/Standards/Generic/Sniffs/Commenting/DocCommentSniff.php

```diff
@@ -183,7 +220,22 @@ public function process(File $phpcsFile, $stackPtr)
 
                 if (preg_match('/^\p{Ll}/u', $tokens[$long]['content']) === 1) {
                     $error = 'Doc comment long description must start with a capital letter';
-                    $phpcsFile->addError($error, $long, 'LongNotCapital');
+
+                    $firstCharacter      = mb_substr($tokens[$long]['content'][0], 0, 1);
+                    $firstCharacterUpper = mb_strtoupper($firstCharacter);
+                    $fix = false;
```

Comment:
The motivation for putting it afterwards was to keep each side of the `if` (ie, `addFixableError()` versus `addError()`) together. I've switched this around now to avoid the need to set this variable here.